packages/advanced-logic/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@requestnetwork/currency": "0.9.0",
+    "@requestnetwork/smart-contracts": "0.29.0",
```

Comment:
my opinion is that we should not add the smart-contracts dependecy to core package like advanced-logic, because of its size.

The bundle size is already a huge issue today, let's not make it worse. 

I'm sure there's a way to revert the dependency and inject the chains information inside the advanced-logic :)